packages/snap/package.json

```diff
@@ -1,20 +1,20 @@
 {
-  "name": "snap-watch-only-snap",
+  "name": "@metamask/snap-watch-only-snap",
   "version": "1.0.0",
-  "description": "A keyring snap for watch-only accounts that integrates with MetaMask accounts.",
+  "description": "Keep an eye on Ethereum addresses or ENS domains right in MetaMask",
   "keywords": [
     "metamask",
     "snap",
     "keyring",
     "account"
   ],
-  "homepage": "https://github.com/k-g-j/snap-simple-keyring-watch-only#readme",
+  "homepage": "https://github.com/metamask/snap-watch-only/packages/snap#readme",
   "bugs": {
-    "url": "https://github.com/k-g-j/snap-simple-keyring-watch-only/issues"
+    "url": "https://github.com/metamask/snap-watch-only/packages/snap/issues"
```

Comment:
same here

Reply:
Can be replaced with: https://github.com/MetaMask/snap-watch-only/issues
addressed here [444afde](https://github.com/MetaMask/snap-watch-only/pull/2/commits/444afde80235c2d9f9425acdb6c24c08515ecc0a)